pkgs/top-level/all-packages.nix

```diff
@@ -36238,7 +36238,10 @@ with pkgs;
 
   xzoom = callPackage ../tools/X11/xzoom {};
 
-  yabai = callPackage ../os-specific/darwin/yabai { };
+  yabai = callPackage ../os-specific/darwin/yabai {
+    inherit (darwin.apple_sdk.frameworks) Cocoa Carbon ScriptingBridge;
+    inherit (darwin.apple_sdk_11_0.frameworks) SkyLight;
+  };
```

Comment:
```suggestion
  yabai = darwin.apple_sdk_11_0.callPackage ../os-specific/darwin/yabai {
    inherit (darwin.apple_sdk.frameworks) Cocoa Carbon ScriptingBridge SkyLight;
  };
```

Reply:
What is the point of `apple_sdk_11.callPackage` if we still end up inheriting all the frameworks manually?

Reply:
Applied your suggestion, saw it uses some different libsystem stuff with the change. Assuming thats benefitial.

Reply:
It is just a thin wrapper around callPackage to make sure all the frameworks work and is required to use the new 11 sdk.